### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by cs-prerna-joshi**
PMDB: https://pmdb.fortinet.com/ProjectManagement/editprojects.php?id=24664 

UTC:
**Get FortiNDR Detection Scenario**
- validated Alerts get created based on FortiNDR Cloud Detection in FortiSOAR.
- validated after running the Get Detection Events PB , events get appended in source data of alert and indicators got created.
- also checked the asset got linked to alert.

### CODE REVIEW

File: info.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            "name": "socSimulator",
+            "type": "solutionpack",
+            "label": "SOC Simulator",
+            "minVersion": null
+        }
+    ],
+    "prerequisite": null,
+    "publisher": "Fortinet",
+    "description": "This FortiSOAR Solution Pack for FortiNDR Cloud offering helps customers to readily orchestrate the investigation of detection.",
+    "help": null,
```
Comment (cs-rohit-khune) [inline info.json]: Please add the doc link: https://github.com/fortinet-fortisoar/solution-pack-fortindr-cloud/blob/release/1.0.0/README.md

File: info.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                "name": "Scenario",
+                "apiName": "scenario"
+            }
+        },
+        "playbooks": [
+            {
+                "name": "02 - Use Case - FortiNDR Cloud"
+            }
+        ],
+        "globalVariables": [
```
Comment (cs-rohit-khune) [inline info.json]: Please remove globaVariables

File: info.json [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    "description": "This FortiSOAR Solution Pack for FortiNDR Cloud offering helps customers to readily orchestrate the investigation of detection.",
+    "help": null,
+    "category": [
+        "Cloud Security"
+    ],
+    "supportInfo": null,
+    "iconLarge": "fsr-icon-large.png",
+    "date": "2023-03-30T09:15:16+00:00",
+    "contents": {
+        "modules": {
```
Comment (cs-rohit-khune) [inline info.json]: Please remove scenario module

File: info.json (hunk 1 above)
Comment (cs-suyog-jinturkar) [inline info.json]: help link to .md file

File: modules/scenario/mmd.json [hunk 4]
```diff
@@ -0,0 +1,306 @@
+{
```
Comment (cs-rohit-khune) [inline modules/scenario/mmd.json]: Please remove scenario mmd.json

File: info.json [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+        {
+            "name": "socSimulator",
+            "type": "solutionpack",
+            "label": "SOC Simulator",
+            "minVersion": null
+        }
+    ],
+    "prerequisite": null,
+    "publisher": "Fortinet",
+    "description": "This FortiSOAR Solution Pack for FortiNDR Cloud offering helps customers to readily orchestrate the investigation of detection.",
```
Comment (cs-suyog-jinturkar) [inline info.json:24]: It will be great if we can add detailed desc here

File: info.json [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+    "help": null,
+    "category": [
+        "Cloud Security"
+    ],
+    "supportInfo": null,
```
Comment (cs-suyog-jinturkar) [inline info.json]: Fortinet Support

File: modules/scenario/mmd.json
Comment (cs-rohit-khune) [inline modules/scenario/mmd.json:1]: Remove this file

Comment (cs-suyog-jinturkar) [inline modules/scenario/mmd.json:1]: We should not include scenario mmd.json here. Any reason for its inclusion in this SP?

File: playbooks/globalVariables.json
Comment (cs-rohit-khune) [inline playbooks/globalVariables.json:1]: This should be an empty list. Please remove the global variable

File: records/scenario/scenario0001.json [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+                    "name": "High Risk Typo Domain Response",
+                    "type": "executePlaybook",
+                    "playbookIRI": "\/api\/3\/workflows\/83dfbd95-2239-4a5f-96bc-b281d473e434",
+                    "preCreateMessage": "Creating demo alert record ",
+                    "postCreateMessage": "",
+                    "waitTimeforNextStep": "0"
+                }
+            ]
+        },
+        "createdAlertsID": "[\"\/api\/3\/alerts\/c8346414-6a30-43f6-bc48-c8675c7b0eb1\"]",
```
Comment (cs-rohit-khune) [inline records/scenario/scenario0001.json]: createdAlertsID value should be null. Please remove alert IRI from the scenario record

**Review by cs-rohit-khune — CHANGES_REQUESTED** [review]

File: playbooks/02 - Use Case - FortiNDR Cloud/Fortinet FortiNDR Cloud > Fetch Detection Events.json [hunk 8]
```diff
@@ -0,0 +1,224 @@
...
+            "uuid": "d9678d1e-3fe4-4079-866f-45841040a2f5"
+        }
+    ],
+    "groups": [],
+    "priority": "\/api\/3\/picklists\/2b563c61-ae2c-41c0-a85a-c9709585e3f2",
+    "uuid": "a871d07b-ea0e-4a0a-a41d-78c35a157864",
+    "owners": [],
+    "isPrivate": false,
+    "deletedAt": null,
+    "recordTags": []
```
Comment (cs-rohit-khune) [inline playbooks/02 - Use Case - FortiNDR Cloud/Fortinet FortiNDR Cloud > Fetch Detection Events.json]: Please add `ManualAction` tag to the playbook

**Review by cs-rohit-khune — APPROVED** [review]

**Review by cs-rohit-khune — APPROVED** [review]
Approved

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**